kalite/settings/base.py

```diff
@@ -215,6 +216,12 @@
         os.mkdir(DEFAULT_DATABASE_PATH)
     
     DEFAULT_DATABASE_PATH = os.path.join(DEFAULT_DATABASE_PATH, 'default.sqlite')
+
+    TT_DATABASE_PATH = os.path.join(USER_DATA_ROOT, "database",)
```

Comment:
I'll be moving this to the /content/khan/ folder, since it's getting downloaded along with the /content/khan/ images, and hence will get moved around with them if the folder is moved -- seems like the best way to ensure the existence of the populated DB aligns with the existence of the corresponding images.